test/utils/browser.js

```diff
@@ -26,36 +48,81 @@ exports.openMusic = function (options) {
   //   "Shuffle artists" is a much better and faster alternative
   options = options || {};
   var url = options.url || 'https://play.google.com/music/listen#/artists';
+  var testName = options.testName || 'Generic test';
```

Comment:
Instead of doing a fallback, can we add an `assert` with an error message?

```js
assert(testName, 'Expected `options.testName` to be provided but it was not. Please provide a name for this test.');
```